telegram/objects/file.py

```diff
@@ -1,15 +1,29 @@
 from .base import BaseObject
 
 class File(BaseObject):
-    def __init__(self,
-        file_id,
-        file_unique_id,
-        file_size = None,
-        file_path = None
-        ):
+    """
+    This object represents a file ready to be downloaded.
 
+    Args:
+        file_id (str): Unique identifier for this file
+
+        file_unique_id (str): Unique identifier for this file, which is supposed to be the same over time and for different bots. Can't be used to download or reuse the file.
+        
+        file_size (int): Optional. File size, if known
+        
+        file_path (str): Optional. File path. Use https://api.telegram.org/file/bot<token>/<file_path> to get the file.
+    """
+
+    __slots__ = (
+        'file_size', 
+        'file_path'
+        )
```

Comment:
`file_size` and `file_path` need to be listed here as well.

Reply:
I thought they are optional,

Reply:
Yes, they are optional as telegram properties, but they still have to be present in python or else we shall have attribute errors when we try accessing them. So basically, every data member of the class must appear in the `__slots__` variable for it to be created in the `__init__` function.

Reply:
Okay, so if I got you right, It should be something like this.

```python
class SomeClass(BaseObject):
         __slots__ = (
         'all properties here'
        )

      def __init__(self, all_properties_as_args):
               self.all_properties_as_args = all_properties_as_args
```

Reply:
It's coming more this
``` python

class TestObject:

    __slots__ = ("a","b","c","d") #can be written in column form.

    def __init__ (self,x,y):
        self.a = x
        self.b = y
        self.c = None
        self.d = None
```
As you see, data members listed in slots are also declared in the init function with the same name. The init arguments are just regular arguments of any function.
I can't explain a lot here, but try reading online about slots to get more info on how they work.